### DISCUSSION TITLE
Blog categories and tags

### DISCUSSION THREAD

**Discussion by andrepatt** [mearashadowfax/ScrewFast#272] [Q&A]
It would be great if blog posts could have tags and categories with the possibility to filter blog posts for those attributes. I'm a beginner, could someone point me out to do that? Or, better, someone has already done that?

Furthermore, a search functionality for blog posts (also for content) or products would be so good.

Btw, awesome template!!!

**Comment by mearashadowfax** [comment]
Hey @andrepatt! You can check out the implementation of tags and search functionality here: https://github.com/mearashadowfax/OpenStove/tree/882c02efd0876d4a5b5180360a602aedd800b6e9/src/pages/recipes
Basically, we filter posts (or any Content Collection) by a key and display the results on a new page. The same logic applies to the search functionality. Hope this helps!
